qemu/tests/multi_disk_random_hotplug.py

```diff
@@ -276,7 +276,15 @@ def _unplug(new_devices, qdev, monitor, prefix=""):
             if device in qdev:  # Some devices are removed with previous one
                 time.sleep(unplug_sleep)
                 unplug_devs.append(device)
-                unplug_outs.append(device.unplug(monitor))
+                try:
+                    output = device.unplug(monitor)
+                except Exception:
+                    # Pass error caused by using cmd '__com.redhat_drive_del'
```

Comment:
Please add comments like:
driver unplug after unplug block device by command 'device_del' in qemu >=2.6, but it not in old qemu. Below update is  compatible with the new version of qemu